pkg/handlers/generic/lifecycle/csi/aws-ebs/handler.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/d2iq-labs/capi-runtime-extensions/common/pkg/k8s/client"
+	"github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AWSEBSConfig struct {
+	defaultsNamespace          string
+	defaultAWSEBSConfigMapName string
+}
+
+func (a *AWSEBSConfig) AddFlags(prefix string, flags *pflag.FlagSet) {
+	flags.StringVar(
+		&a.defaultsNamespace,
+		prefix+".defaultsNamespace",
+		corev1.NamespaceDefault,
+		"namespace of the ConfigMap used to deploy AWS EBS CSI driver",
+	)
+	flags.StringVar(
+		&a.defaultAWSEBSConfigMapName,
+		prefix+".aws-ebs-provider-configmap-name",
+		"aws-ebs-csi-crs-cm",
```

Comment:
More consistent with NFD and Calico where we don't include "cm" or "crs"
```suggestion
		"aws-ebs-csi",
```